backend/berkeleytime/config/semesters/spring2024.py

```diff
@@ -0,0 +1,41 @@
+"""Configurations for Spring 2024."""
+# from berkeleytime.config.finals.semesters.spring2024 import *
+
+import datetime
+
+CURRENT_SEMESTER = 'spring'
+CURRENT_YEAR = '2024'
+CURRENT_SEMESTER_DISPLAY = 'Spring 2024'
+
+# SIS API Keys
+SIS_TERM_ID = 2242
+
+TELEBEARS = {
+    'phase1_start': datetime.datetime(2023, 10, 24),
+    'phase1_end': datetime.datetime(2023, 11, 5),
+    'phase2_start': datetime.datetime(2023, 11, 22),
+    'phase2_end': datetime.datetime(2023, 1, 7),
+    'adj_start': datetime.datetime(2023, 1, 8),
```

Comment:
Should be 2024.

Reply:
also i think phase1 and phase2 start should be 10/16 and 11/14 respectively